notebooks/en/index.md

```diff
@@ -7,11 +7,11 @@ applications and solving various machine learning tasks using open-source tools
 
 Check out the recently added notebooks:
 
+- [Efficient Online Training with GRPO and vLLM in TRL](grpo_vllm_online_training)
 - [Fine-tuning LLMs for Function Calling with the xLAM Dataset](function_calling_fine_tuning_llms_on_xlam)
 - [Post training an VLM for reasoning with GRPO using TRL](fine_tuning_vlm_grpo_trl)
 - [TRL GRPO Reasoning with Advanced Reward](trl_grpo_reasoning_advanced_reward)
 - [Fine-Tuning a Vision Language Model with TRL using MPO](fine_tuning_vlm_mpo)
-- [Fine tuning a VLM for Object Detection Grounding using TRL](fine_tuning_vlm_object_detection_grounding)
```

Comment:
@sergiopaniego why was this removed? feel free to merge it once this one's addressed!